changelog 12/11 (#440)
lastmile-ai/aiconfig: CHANGELOG.md

# Changelog

> ## Python Version 1.1.5 (2023-12-11)

Reviewer: 1.1.5

## NPM Version 1.0.7 (2023-12-11)

Full Changelog:

### Features

> - **python-sdk** Evaluation harness with AIConfig. Supports text input/output evaluation with native AIConfig Interface as well as an integration with Promptfoo. (Link to ReadME)

Reviewer: ```suggestion
- **python-sdk** Evaluation harness for AIConfig. Supports text input/output evaluation with native AIConfig Interface ([tutorial](https://github.com/lastmile-ai/aiconfig/blob/9669b9e5614d5b1b0603ddf3ec61556cd79bfc14/python/src/aiconfig/eval/examples/travel/travel_eval.ipynb)) as well as an integration with Promptfoo ([tutorial](https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/eval/README.md#usage-guide---promptfoo-integration)). See [README](https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/eval/README.md) for details
```

- **python-sdk:** Support for PaLM Text as a core model Parser ([222bf6e](https://github.com/lastmile-ai/aiconfig/pull/348/commits/222bf6edf5c615b3b9b5f58b7fd5c829b2884d1b))
- **typescript-sdk:** Support for PaLM Text as a core model parser ([222bf6e](https://github.com/lastmile-ai/aiconfig/pull/348/commits/222bf6edf5c615b3b9b5f58b7fd5c829b2884d1b))
- - Link to video demo

### Bug Fixes

- **python-extension** HuggingFace Transformers Extension: Fixed bug where we're not properly appending outputs for multiple return sequences (https://github.com/lastmile-ai/aiconfig/pull/411)

### Extensions

> - **python-extension:** Extension for LLama-Guard using pytorch ([222bf6e](https://github.com/lastmile-ai/aiconfig/pull/348/commits/222bf6edf5c615b3b9b5f58b7fd5c829b2884d1b))

Reviewer: Also include not defaulting to gpt2 for model name
